### PR TITLE
react1 week1 hw TodoList

### PR DESCRIPTION
**Pull request by mehmetnuricavdar**

### CODE REVIEW

File: react/react1/week1/todo-list/src/components/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import TodoHeader from "./TodoHeader";
+import TodoItem from "./TodoItem";
+
+function TodoList() {
+  return (
+    <div>
+      <TodoHeader />
+      <TodoItem description="Get out of bed" deadline="Wed Sep 13 2017" />
```
Comment (annamourasobral) [inline react/react1/week1/todo-list/src/components/TodoList.js]: You definitely did the job but would be interesting to have these items' descriptions and deadlines in objects inside an array that you could map and pass this information as props.

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hey, @mehmetnuricavdar 
Congrats on fulfilling this task so quickly! 🚀

File: react/react1/week1/todo-list/src/components/TodoList.js (hunk 1 above)
Comment (mehmetnuricavdar) [reply react/react1/week1/todo-list/src/components/TodoList.js]: Yes, that's a great idea!  Thank you so much for reviewing my hw!

**Comment by mehmetnuricavdar** [comment]
> Hey, @mehmetnuricavdar Congrats on fulfilling this task so quickly! 🚀

Thank you so much for reviewing my hw!

**Pull request merged**